### DISCUSSION TITLE
Resume image playback/shuffle playback

### DISCUSSION THREAD

**Discussion by aurel51** [Geektoolkit/Dynaframe3#140] [Q&A]
Hello

I'm using the stable version of Dynaframe (2.13), it's already a great job and it replaces the standard photo frames very well. I have some questions about some features that I don't seem to have seen or managed to configure. Is there a shuffle function? When I turn off the raspebrry pi and turn it back on Dynaframe always restarts from the first photo, is there a function to pick up where it left off?

Thanks again for this great development !

**Comment by Geektoolkit** [comment]
There is a shuffle button in the controls portion of the UI, however it works better in 2.18 (which I really need to promote to stable).  in 2.13 it can get less 'shuffled' if you have large lists. However if you turn it off and then on, it doesn't continue where you left off. It would just reshuffle on startup and go from there.  I usually keep my raspberry pi on on mine and just turn off the montior, or use the webUI to turn the monitor signal off (which turns off most modern monitors).
